app/main.py

```diff
@@ -2,0 +1,17 @@
+from app.errors import (NotWearingMaskError, VaccineError)
+
+
+def go_to_cafe(friends: list, cafe: str) -> str:
```

Comment:
The type hint for the `cafe` parameter should specify the `Cafe` class instead of `str`. Update the type hint to `cafe: Cafe` to reflect the actual type being used.